Structure_Engine/Modify/AddRebarIntent.cs

```diff
@@ -38,24 +38,26 @@ public static partial class Modify
         /**** Public Methods                            ****/
         /***************************************************/
 
-        [Description("Adds Reinforcement to a ConcreteSection. Any previous Reinforcement will be kept.")]
-        [Input("section", "The concrete section to add Reinforcement to.")]
-        [Input("reinforcement", "The collection of Reinforcement to add to the ConcreteSection.")]
-        [Output("concSection", "The ConcreteSection with additional Reinforcement.")]
-        public static ConcreteSection AddReinforcement(this ConcreteSection section, IEnumerable<IBarReinforcement> reinforcement)
+        [Description("Adds RebarIntent to a ConcreteSection. Any previous RebarIntent will be kept.")]
+        [Input("section", "The concrete section to add RebarIntent to.")]
+        [Input("rebarIntent", "The RebarIntent to add to the ConcreteSection.")]
+        [Output("concSection", "The ConcreteSection with RebarIntent.")]
+        public static ConcreteSection AddRebarIntent(this ConcreteSection section, BarRebarIntent rebarIntent)
```

Comment:
Would revert most of the changes in this method, as the intention was to be able to add more reinforcement to a section that already potentially had some reinforcement on it.

Would keep signature and most of the descriptions as before, but change the behaviour to instead add reinforcement to the RebarIntent obejct owned by the concrete section.